Dockerfile-alpine.template

```diff
@@ -15,9 +15,9 @@ RUN set -ex \
     && apk add --no-cache \
        gettext \
        postgresql-client=$PG_VERSION \
-    && apk add --no-cache --repository http://dl-cdn.alpinelinux.org/alpine/edge/main \
```

Comment:
It would be valuable to surface in the PR body what's going on here:

It looks like the version of `gdal` in the Alpine Linux package repositories used to depend on `libressl2.7-libcrypto`, which is available in the `v3.8/main` branch.

The default set of repositories configured in the root container image are:

```bash
/ # cat /etc/apk/repositories
http://dl-cdn.alpinelinux.org/alpine/v3.8/main
http://dl-cdn.alpinelinux.org/alpine/v3.8/community
```

If we want to look for packages from another repository, that repository needs to be explicitly stated with `--repository`.

The `gdal` package was updated to depend on `libcrypto1.1`, which is only available in the `edge/main` branch.

For more information on Alpine Linux package management, see: https://wiki.alpinelinux.org/wiki/Alpine_Linux_package_management

Also, small formatting nitpick:

```diff
diff --git a/Dockerfile-alpine.template b/Dockerfile-alpine.template
index 0f5e3dc..93615fe 100644
--- a/Dockerfile-alpine.template
+++ b/Dockerfile-alpine.template
@@ -16,8 +16,7 @@ RUN set -ex \
        gettext \
        postgresql-client=$PG_VERSION \
     && apk add --no-cache --repository http://dl-cdn.alpinelinux.org/alpine/edge/main \
-        libressl2.7-libcrypto \
-    && apk add --no-cache --repository http://dl-cdn.alpinelinux.org/alpine/edge/testing \
+       --repository http://dl-cdn.alpinelinux.org/alpine/edge/testing \
        gdal \
     && pip install --no-cache-dir -r /tmp/requirements.txt \
     && apk del .build-deps \
```